src/pages/main/MainPage.tsx

```diff
@@ -49,7 +49,7 @@ export default function MainPage() {
     fileTree,
     activePanel,
     autoSave,
-    formatCode,
+    // formatCode, // TODO: it is temorally commented - in future it is possibly used or removed
```

Comment:
just remove it, in future we can add it if we will really need it

Reply:
++, removed